Review during July sprint (#75)
carpentries-incubator/managing-computational-projects: _episodes/01-introduction.md

As pictured below, in order to apply these tools, researcher first need to understand and apply the building blocks of data science, especially research data management, collaborative working and project management.

> In what aspects of your projects do you already apply computational and statistical approaches?
> Do you consider data science relevant for your project? Why/Why not? 
> 
{: .discussion}

The [**Data Science for Biomedical Scientists**](https://github.com/alan-turing-institute/data-training-for-bioscience) project helps address this need in training by equipping experimental biomedical scientists with essential computational skills.
In all the resources developed within this project, we consistently emphasise how computational and data science approaches can be applied while ensuring reproducibility, collaboration and transparent reporting.

> **The goal is to maintain the highest standards of research practice and integrity**.
>
{: .testimonial}

> In this training material for **Managing Open and Reproducible Computational Projects**, we discuss essential practices for computational reproducibility required for carrying out meaningful analyses of research datasets through data exploration, processing, visualisation and communication. 

Reviewer: not sure about the title, especially the "open" part. it may scare people.

Author: we can totally drop open and reproducible both
```suggestion
In this training material for learning how to manage computational projects, we discuss essential practices for computational reproducibility required for carrying out meaningful analyses of research datasets through data exploration, processing, visualisation and communication. 
```
